drivers/wifi/nrf700x/zephyr/src/zephyr_wpa_supp_if.c

```diff
@@ -452,6 +452,16 @@ int wifi_nrf_wpa_supp_scan2(void *if_priv, struct wpa_driver_scan_params *params
 	scan_info.scan_mode = 0;
 	scan_info.scan_reason = SCAN_CONNECT;
 
+	/* Copy extra_ies */
+	if (params->extra_ies_len && params->extra_ies_len <= IMG_MAX_IE_LEN) {
```

Comment:
Change IMG_MAX_IE_LEN to NRF_WIFI_MAX_IE_LEN

Reply:
These are not introduced in this PR, and also present in common header files with RPU, so, left it untouched, once we clean those header files then we can do these changes.